app/assets/javascripts/application.js

```diff
@@ -432,7 +432,7 @@ function setPowerState(item, status){
       $('#loading_power_state').remove();
     }
   }else{
-    $('#loading_power_state').text(_('Unknown power state'))
+    $('#loading_power_state').text(__('Unknown power state'))
```

Comment:
regardless of this PRs fate, this is a good catch and should be fixed separately.